examples/next-openai-pages/app/api/file/route.ts

```diff
@@ -0,0 +1,59 @@
+// https://vercel.com/docs/storage/vercel-blob/client-upload#create-a-client-upload-route
+
+import { handleUpload, type HandleUploadBody } from '@vercel/blob/client';
+import { NextResponse } from 'next/server';
+
+export async function POST(request: Request): Promise<NextResponse> {
```

Comment:
maybe add a short comment above that explains what this route does?

Reply:
done!